### PR TITLE
feat: support com_long_data

### PR DESCRIPTION
**Pull request by dk-lockdown**
ref: https://github.com/cectc/dbpack/issues/<issueID>

### CODE REVIEW

**Comment by adofsauron** [comment]
LGTM

File: pkg/listener/mysql.go [hunk 1]
```diff
@@ -780,8 +781,22 @@ func (l *MysqlListener) ExecuteCommand(ctx context.Context, c *mysql.Conn, data
 			l.stmts.Delete(stmtID)
 		}
 	case constant.ComStmtSendLongData: // no response
-		values := make([]byte, len(data))
-		copy(values, data)
+		if len(data) < 6 {
```
Comment (bohehe) [inline pkg/listener/mysql.go:829]:
```suggestion
		if len(data) < minPKLen {
```

File: pkg/listener/mysql.go [hunk 2]
```diff
@@ -780,8 +781,22 @@ func (l *MysqlListener) ExecuteCommand(ctx context.Context, c *mysql.Conn, data
...
+		stmtID := int(binary.LittleEndian.Uint32(data[0:4]))
+
+		si, ok := l.stmts.Load(stmtID)
+		if !ok {
+			return errors.Errorf("statement not found, statement id: %v", stmtID)
+		}
+		stmt := si.(*proto.Stmt)
+
+		paramID := int(binary.LittleEndian.Uint16(data[4:6]))
+		parameterID := fmt.Sprintf("v%d", paramID+1)
```
Comment (bohehe) [inline pkg/listener/mysql.go:842]: why `v%d`, not `%d`?

**Review by bohehe — APPROVED** [review]

File: pkg/listener/mysql.go (hunk 2 above)
Comment (dk-lockdown) [reply pkg/listener/mysql.go:842]: Same as packet/mysql.go
https://github.com/CECTC/dbpack/blob/be37d88f356e807d90fe57ffe17296752f65b7d5/pkg/packet/mysql.go#L99

File: pkg/listener/mysql.go (hunk 1 above)
Comment (dk-lockdown) [reply pkg/listener/mysql.go:829]: long data packet length must be greater than 6, some others can be less than 6.

**Pull request merged**